llama_index/evaluation/retrieval/evaluator.py

```diff
@@ -83,8 +86,14 @@ async def _aget_retrieved_ids(
                 text_nodes.append(node)
 
         if mode == "text":
-            return [node.node_id for node in text_nodes]
+            return (
+                [node.node_id for node in text_nodes],
+                [node.text for node in text_nodes],
+            )
         elif mode == "image":
-            return [node.node_id for node in image_nodes]
+            return (
+                [node.node_id for node in image_nodes],
+                [node.text for node in image_nodes],
+            )
```

Comment:
very nit: aren't these the same?

Reply:
subtle difference: in that one uses `text_nodes` and the other uses `image_nodes`. We could maybe make it more succinct and just do `nodes = text_nodes` or `nodes=image_nodes`, then just do list comprehension once outside the if/else block. But not sure that its that much better. Since this is very nit, I hope its okay I just leave it as is. :)